### ISSUE TITLE
The `Sample` no longer works with 3.43.2.0

### ISSUE THREAD

**Issue by pzygielo** [xerial/sqlite-jdbc#991]
**Describe the bug**
The `Sample` no longer works with 3.43.2.0.

**To Reproduce**
```
$ java -cp sqlite-jdbc-3.43.2.0.jar:. Sample 
No suitable driver found for jdbc:sqlite:sample.db
```

**Expected behavior**
Like for the version before:
```
$ java -cp sqlite-jdbc-3.43.0.0.jar:. Sample 
name = leo
id = 1
name = yui
id = 2
```

**Logs**
If applicable, provide logs.

**Environment (please complete the following information):**
 - OS: linux (6.5.6-200.fc38.x86_64)
 - CPU architecture: [x86_64]
 - sqlite-jdbc version [3.43.2.0]

**Additional context**

**Comment by pzygielo** [comment]
One now needs to add `slf4j-api` to CP:
```
$ java -cp slf4j-api-2.0.9.jar:sqlite-jdbc-3.43.2.0.jar:. Sample 
SLF4J: No SLF4J providers were found.
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See https://www.slf4j.org/codes.html#noProviders for further details.
name = leo
id = 1
name = yui
id = 2
```

**Comment by gotson** [comment]
🎉 This issue has been resolved in `3.43.2.1` ([Release Notes](https://github.com/xerial/sqlite-jdbc/releases/tag/3.43.2.1))
